sdk-docs/reference/npm-libraries.md

```diff
@@ -0,0 +1,59 @@
+
+| Library | Usage |
+| --- | --- |
+| [@trackunit/css-core](https://www.npmjs.com/package/@trackunit/css-core) | Includes all the Trackunit css base setup and PostCSS & Tailwind configuration. |
+| [@trackunit/css-components](https://www.npmjs.com/package/@trackunit/css-components) | Includes all the css classes for [Trackunits component library](https://developers.trackunit.com/page/ui-components). |
+| [@trackunit/css-component-tokens](https://www.npmjs.com/package/@trackunit/css-component-tokens) | Exposes the Trackunit Component Tokens Plugin. |
+| [@trackunit/css-tailwind](https://www.npmjs.com/package/@trackunit/css-tailwind) | Exposes the Trackunit Tailwind configuration. |
+| [@trackunit/css-tailwind-custom-properties-plugin](https://www.npmjs.com/package/@trackunit/css-tailwind-custom-properties-plugin) | Exposes a plugin used to convert the variables property in a Tailwind configuration to css custom properties. |
+| [@trackunit/tailwind-styled-components](https://www.npmjs.com/package/@trackunit/tailwind-styled-components) | Is a wrapper for [tailwind-styled-components](https://www.npmjs.com/package/tailwind-styled-components) Exposing a small utility for nested inline intellisense and some typing fixes. |
+| [@trackunit/ui-icons](https://www.npmjs.com/package/@trackunit/ui-icons) | Used by the Icon component in [@trackunit/react-components](https://www.npmjs.com/package/@trackunit/react-components) |
+| [@trackunit/ui-design-tokens](https://www.npmjs.com/package/@trackunit/ui-design-tokens) | TODO. |
+
+# React Libraries
+The React libraries do all the plumbing for you in various scenarios. For example when you need to interact with the GraphQL API or you want to get a list of custom fields and display React components without needing to handle state variables yourself.
+| Library | Usage |
+| --- | --- |
+| [@trackunit/react-core-hooks](https://www.npmjs.com/package/@trackunit/react-core-hooks) | Contains core React hooks used for getting data from our context providers inside Iris apps. |
+| [@trackunit/react-components](https://www.npmjs.com/package/@trackunit/react-components) | The home of Trackunits public UI library. |
+| [@trackunit/react-graphql-tools](https://www.npmjs.com/package/@trackunit/react-graphql-tools) | Provides tools for auto-generating hooks to interface our GraphQL API through React code. |
+| [@trackunit/react-core-contexts](https://www.npmjs.com/package/@trackunit/react-core-contexts) | Defines core context providers. TODO: brief description of what these providers are. |
+| [@trackunit/react-core-contexts-test](https://www.npmjs.com/package/@trackunit/react-core-contexts-test) | Contains testing utilities for [@trackunit/react-core-contexts](https://www.npmjs.com/package/@trackunit/react-core-contexts). |
+| [@trackunit/react-core-contexts-api](https://www.npmjs.com/package/@trackunit/react-core-contexts-api) | Contains reusable React types. TODO: elaborate slightly. |
```

Comment:
Types fro the providers

Reply:
Could you elaborate?

Reply:
`react-core-contexts-api` holds the types for the providers and contexts in `react-core-contexts`.
This is what allows the use of the same hooks both inside the Iframe and Outside (this sentence is not relevant for 3. party i think)

Reply:
More info in this 2 min video: https://www.dropbox.com/s/d4cqil17yt40za3/Host%20Providers%20and%20App%20Providers%20%282min%29.mp4?dl=0